src/pages/login/Login.tsx

```diff
@@ -0,0 +1,52 @@
+import { Button, Divider, Form, Input } from 'antd';
+import React from 'react';
+import { useDispatch } from 'react-redux';
+
+import { showDrawer } from '../../redux-features/common';
+import styles from './login.module.scss';
+
+const Login: React.FC = () => {
+  const dispatch = useDispatch();
+
+  const navigateToSignup = () => {
+    // e.preventDefault();
+    dispatch(showDrawer('signup'));
+  };
+
+  return (
+    <div className={styles.loginWrapper}>
+      <Form
+        layout="vertical"
+        // form={form}
+        // onValuesChange={onFormLayoutChange}
+      >
+        <Form.Item label="Email id">
+          <Input placeholder="Enter your email id" size="large" />
+        </Form.Item>
+        <Form.Item label="Password">
+          <Input placeholder="Enter your password" size="large" />
+        </Form.Item>
+        <Form.Item>
+          <Button type="primary" size="large" className={styles.formBtn}>
+            Submit
+          </Button>
+        </Form.Item>
+        <Divider plain>OR</Divider>
+        <Form.Item>
+          <Button type="primary" size="large" className={styles.formBtn}>
+            Login via google
+          </Button>
+        </Form.Item>
+        <Divider orientation="right" plain>
+          {"Don't have an account?"}
```

Comment:
need not wrap in brackets

Reply:
Done to avoid using **&quest** for question mark in plain text